src/components/calcite-dropdown-group/calcite-dropdown-group.tsx

```diff
@@ -6,14 +6,15 @@ import {
   h,
   Host,
   Listen,
-  Prop
+  Prop,
```

Comment:
Nitpick - are we a trailing comma family now 👨‍👧‍👦?

Reply:
It's the auto-formatting in action. 🎩🐇✨  It's the default now in [Prettier 2.0](https://prettier.io/blog/2020/03/21/2.0.0.html).

Reply:
Ah, cool, we should update our `prettierrc` to match https://github.com/Esri/calcite-components/blob/4ee45abba419aba70e60297c6d51671d7818b70d/prettierrc.json#L9 🥇